java/basic_app/README.md

```diff
@@ -15,16 +15,24 @@ The app will be able to register it in AppsFlyer's dashboard, create real OneLin
 ## How to use this app?
 1. Clone the repository
 2. Change the package to your own package name. This is the package name that will be used AppsFlyer's dashboard.
-3. Get your AppsFlyer Dev Key using [these instructions][get_af_devkey].
-4. Create the file `java/simple_deep_link_pass_data/app/src/main/java/com/appsflyer/onelink/appsflyeronelinkbasicapp/AppsFlyerConstants.java` with the following content:
+3. Run the app, preferebly on a real device, as emulators has a few issues.
+
+## How to make the app your own?
+> ‼️ Important 
+> This app runs by default with a devkey of a demo account.
+> If you would like to see the app's data in the AppsFlyer dashboard, you need to create a demo app in your AppsFlyer account and follow the instructions below.
+
+1. Get your AppsFlyer Dev Key using [these instructions][get_af_devkey].
+2. Create the file `java/simple_deep_link_pass_data/app/src/main/java/com/appsflyer/onelink/appsflyeronelinkbasicapp/AppsFlyerConstants.java` with the following content:
```

Comment:
The file already exist since we pushed it to GitHub. We can ask the developers to add it to .gitignore .